### DISCUSSION TITLE
Terria v7 to v8 Pre-populate layers

### DISCUSSION THREAD

**Discussion by anujdivesh** [TerriaJS/terriajs#5836] [Q&A]
Hi All, I recently transitioned from Terria v7 to v8. In my Terriajs v7, i used to per-populate layers as the URL was opened, by using "zoomOnEnable": true feature. 

Any leads how I can do this in v8.

Thanks in advance.

**Comment by zoran995** [comment]
I don't see this implemented in v8, just created a #5882 for it
